core/src/main/java/org/mskcc/cbio/portal/model/SampleListCategory.java

```diff
@@ -52,6 +52,7 @@
     ALL_CASES_WITH_MUTATION_AND_CNA_DATA("all_cases_with_mutation_and_cna_data"),
     ALL_CASES_WITH_MUTATION_AND_CNA_AND_MRNA_DATA("all_cases_with_mutation_and_cna_and_mrna_data"),
     ALL_CASES_WITH_GSVA_DATA("all_cases_with_gsva_data"),
+    ALL_CASES_WITH_SV_DATA("all_cases_with_structural_variant_data"),
```

Comment:
Hi @khzhu, we are using `all_cases_with_sv_data` as the category name instead of `all_cases_with_structural_variant_data`.